cgp-utils_config.md

```diff
@@ -0,0 +1,114 @@
+# Configuration
+
+The [cpg-utils library](https://github.com/populationgenomics/cpg-utils) ([pypi](https://pypi.org/project/cpg-utils/)) contains a streamlined config management tool. This config management is used by the seqr-loader pipeline, but is useful in projects at any scale.
+
+This configuration tool uses one or more `TOML` files, and creates a dictionary of key-value attributes which can be accessed at any point, without explicitly passing a configuration object.
+
+## TOML
+
+[Tom's Obvious, Minimal Language](https://toml.io/en/) is a config file format designed to be easily human readable and writeable, with clear data structures. Sections are delineated using bracketed headings, and key-value pairs are defined using `=` syntax, e.g. :
+
+```toml
+global_key = "value"
+
+[heading_1]
+name = "Luke Skywalker"
+age = 53
+
+[heading_1.subheading]
+occupation = ["Jedi", "Hermit", "Force Ghost"]
+```
+
+will be digested into the simple dictionary:
+
+```python
+{
+    'global_key': 'value',
+    'heading_1': {
+        'name': 'Luke Skywalker',
+        'age': 53,
+        'subheading': {
+            'occupation': ["Jedi", "Hermit", "Force Ghost"]
+        }
+    }
+}
+```
+
+## Using cpg-utils.config
+
+To use the `cpg-utils.config` functions, import these two key methods into any script:
+
+```python
+from cpg_utils.config import get_config, set_config_paths
```

Comment:
You generally don't need to call `set_config_paths` as analysis-runner does that for you